docs/src/@primer/gatsby-theme-doctocat/components/head.js

```diff
@@ -18,8 +18,9 @@ function Head(props) {
       <meta property="og:image" content={siteMetadata.imageUrl} />
       <meta property="twitter:card" content="summary_large_image" />
       <link href="https://unpkg.com/@primer/css-next@canary/dist/primer.css" rel="stylesheet" />
+      <script src="https://unpkg.com/@primer/view-components@0.0.26/app/assets/javascripts/primer_view_components.js"></script>
```

Comment:
not sure how I feel about it since we'll have to keep updating the version here.
it also doesn't fix the local development issue

Reply:
Changed it to latest so we dont have to update. Doesn't fix local development but its a step in the right direction :smiley_cat: